packages/groq-builder/src/commands/select.test.ts

```diff
@@ -186,9 +186,9 @@ describe("select", () => {
     it("should fail with invalid data", async () => {
       await expect(() => executeBuilder(qSelect, invalidData)).rejects
         .toThrowErrorMatchingInlineSnapshot(`
-        "2 Parsing Errors:
+        [ValidationErrors: 2 Parsing Errors:
```

Comment:
It appears that the snapshot changed here to include the ErrorType. At this point it is unclear why. Will investigate.

Reply:
That seems fine to me. It's a better snapshot.